src/Sylius/Behat/Page/Shop/Checkout/ThankYouPage.php

```diff
@@ -15,17 +15,34 @@
 
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
  */
 class ThankYouPage extends SymfonyPage implements ThankYouPageInterface
 {
     /**
      * {@inheritdoc}
      */
-    public function hasThankYouMessageFor($name)
+    public function hasThankYouMessage()
     {
-        $thankYouMessage = $this->getElement('thank you message')->getText();
+        $thankYouMessage = $this->getElement('thank_you')->getText();
 
-        return false !== strpos($thankYouMessage, sprintf('Thank you %s', $name));
```

Comment:
Don't we want to leave thank you message with customer name/email? It will be useful for thank you page for guest, but I think it would also be nice to have it on thank you page for logged in customer.